nucypher/cli/commands/stake.py

```diff
@@ -811,14 +811,26 @@ def collect_reward(general_config,
     if staking_reward:
         # Note: Sending staking / inflation rewards to another account is not allowed.
         reward_amount = NU.from_nunits(STAKEHOLDER.calculate_staking_reward())
+        if reward_amount == 0:
+            emitter.echo(NO_TOKENS_TO_WITHDRAW, color='red')
+            raise click.Abort
+
         emitter.echo(message=COLLECTING_TOKEN_REWARD.format(reward_amount=reward_amount))
+
+        if not force and STAKEHOLDER.non_withdrawable_stake() == 0 and STAKEHOLDER.mintable_periods() > 0:
```

Comment:
`== 0 and`?

Reply:
what do you mean by groups?

Reply:
My eyes naturally read `A == 0 and STAKEHOLDER` - adding groups with `(` and `)`.

Reply:
`if not force and (STAKEHOLDER.non_withdrawable_stake() == 0 and STAKEHOLDER.mintable_periods() > 0):` ?

Reply:
Haha We're crossing into style preferences here but even better is to name the booleans.

Reply:
if I'll just move to variables short circuit won't work and `mintable_periods` will be called always
how to do that properly?

Reply:
✔️

Reply:
Yeah - I hear ya - there's a balance to strike.  In this case, I think an additional level of nesting is okay.